sdk/storage/Azure.Storage.Queues/tests/QueueUriBuilderTests.cs

```diff
@@ -22,7 +22,7 @@ public QueueUriBuilderTests(bool async)
         public void QueueUriBuilder_RegularUrl_AccountTest()
         {
             // Arrange
-            var uriString = "https://account.core.queue.windows.net?comp=list";
+            var uriString = "https://account.queue.core.windows.net?comp=list";
```

Comment:
I had to make correction to these tests here, since it's a copy and paste typo where ".core" was before ".queue".